src/data/list/count.lean

```diff
@@ -0,0 +1,175 @@
+/-
+Copyright (c) 2014 Parikshit Khanna. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Parikshit Khanna, Jeremy Avigad, Leonardo de Moura, Floris van Doorn, Mario Carneiro
```

Comment:
Mario for 8f4327ae0c45ed36037beb1449f8285cae1ebe9c and then it goes to core Lean.